### PR TITLE
[near-contract-helper] construct useful client repsonse from /account

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -138,26 +138,36 @@ router.get('/account/:name', async ctx => {
...
         amount: newAccountAmount,
-        public_key: hardcodedKey.public_key,
+        public_key: accountKey.public_key,
     };
 
-    const resp = await submit_transaction_rpc(client, "create_account", createAccountParams);
-    checkError(ctx, resp);
-    ctx.body = resp.result;
+    const transactionResponse = await submit_transaction_rpc(client, "create_account", createAccountParams);
+    checkError(ctx, transactionResponse);
```
Comment (vgrichina) [inline app.js]: This method doesn't take `ctx` anymore:
https://github.com/nearprotocol/near-contract-helper/pull/7/files

Comment (vgrichina) [reply app.js]: `submit_transaction_rpc` also changed.

File: app.js [hunk 2]
```diff
@@ -138,26 +138,36 @@ router.get('/account/:name', async ctx => {
...
+
+    // transactionResponse does not contain useful information, so construct a custom response
+    // TODO: is it fine to return private key here? 
+    const clientResponse = {
+        account_name: newAccountName,
```
Comment (vgrichina) [inline app.js]: let's use camelCase in our responses.

File: app.js [hunk 3]
```diff
@@ -138,26 +138,36 @@ router.get('/account/:name', async ctx => {
...
     };
 
-    const resp = await submit_transaction_rpc(client, "create_account", createAccountParams);
-    checkError(ctx, resp);
-    ctx.body = resp.result;
+    const transactionResponse = await submit_transaction_rpc(client, "create_account", createAccountParams);
+    checkError(ctx, transactionResponse);
+
+    // transactionResponse does not contain useful information, so construct a custom response
+    // TODO: is it fine to return private key here?
```
Comment (vgrichina) [inline app.js]: we don't need to return it, we need to record it as signing is handled on our side.

File: app.js [hunk 4]
```diff
@@ -138,26 +138,36 @@ router.get('/account/:name', async ctx => {
 
 /**
  * Create a new account. Generate a throw away account id (UUID).
```
Comment (vgrichina) [inline app.js]: Let's better do API docs in a separate doc using https://apiblueprint.org
I think ultimately it's far more clear if you can see possible example responses, etc

Comment (janedegtiareva) [reply app.js]: Agreed. It will be easier to do more robust docs from these breadcrumbs though, so let's keep them for now.

File: app.js [hunk 5]
```diff
@@ -177,21 +177,32 @@ router.get('/account/:name', async ctx => {
...
     };
 
-    ctx.body = await submitTransaction("create_account", createAccountParams);
+    const transactionResponse = await submitTransaction("create_account", createAccountParams);
+    checkError(transactionResponse);
+
+    // transactionResponse does not contain useful information, so construct a custom response
+    // TODO: record key info
+    const clientResponse = {
+        accountName: newAccountName,
```
Comment (vgrichina) [inline app.js]: let's also return `accountHash`, cause now we need to use hashes when passing params to contract method calls.

**Pull request merged**